### ISSUE TITLE
Validation blocked - Cannot read properties of undefined (reading '.js')

### ISSUE THREAD

**Issue by playniuniu** [NomicFoundation/hardhat-vscode#305]
I create a new template project with command: npx hardhat and select typescript version.

When I open the .sol file with vscode, it has this problem: Validation blocked Cannot read properties of undefined (reading '.js')

My Create Procedure:

npm init -y
npx hardhat
npx hardhat compile

My environment:

OS Version: macOS. 12.6.1 (Apple Silicon M1)
Hardhat Version: 2.12.3
VSCode Version:  1.73.1
Hardhat Solidity Plugin Version: 0.6.3

**Comment by playniuniu** [comment]
My hardhat.config.ts is below

```typescript
import { HardhatUserConfig } from "hardhat/config";
import "@nomicfoundation/hardhat-toolbox";

const config: HardhatUserConfig = {
  solidity: "0.8.17",
};

export default config;
```

**Comment by antico5** [comment]
I couldn't reproduce the issue on Linux. The "Validation blocked (...)" message means the local hardhat module couldn't run the compilation job. Can you double check that a) you are opening the correct project folder on vs code and b) you are able to run `npx hardhat compile` on such project?

**Comment by playniuniu** [comment]
I double check the procedure and have the same problem. I also install an arm64 debian linux, with newest vscode and hardhat plugin, and have the same problem.

**Comment by kanej** [comment]
@playniuniu I haven't been able to reproduce this issue.

Do you get the same issue if you create a javascript sample project?

**Comment by vladyan18** [comment]
I have the same problem in correct project. The error does not occur in version 0.6.1, occurs in 0.6.2 and newer.

**Comment by vladyan18** [comment]
I tried cloning the extension repository and running [locally](https://github.com/NomicFoundation/hardhat-vscode/blob/development/CONTRIBUTING.md#running-locally). The error does not appear.

**Comment by playniuniu** [comment]
@kanej I can reproduce the problem in both typescript and javascript version sample project.

**Comment by antico5** [comment]
We found the issue: the extension is not working properly when solcJs is being used (likely on arm64 cpus) due to a problem with bundling. We'll address this problem asap

**Comment by daramir** [comment]
> @daramir with version `0.6.3` you are still seeing no `inline code validation from the compiler`?

This is correct @kanej 

> The `textDocument/didChange` error we think is unrelated (it occurs in setups where validation is working).

Thanks. I only mentioned it because on 0.5.5 it only comes up once, whereas on 0.6.3 it comes up multiple times.

Good to hear you've found the issue @antico5

**Comment by daramir** [comment]
@kanej  @antico5 , sorry to bother folks, I'm still having the `no inline code validation from the compiler` issue when using version `0.6.4` (current)

Also still seeing 
```text
Validation blocked - Cannot read properties of undefined (reading '.js')
```
in the bottom right status bar icon next to "solidity".

How can I help in providing a more accurate assessment or the root cause?

**Comment by antico5** [comment]
Sorry to hear that! @playniuniu @vladyan18 could you verify that you still have the issue with 0.6.4? Then we can continue investigating

**Comment by vladyan18** [comment]
> Sorry to hear that! @playniuniu @vladyan18 could you verify that you still have the issue with 0.6.4? Then we can continue investigating

Yes

btw: AMD Ryzen 7 4800H

**Comment by antico5** [comment]
@vladyan18 @daramir thanks for reporting, I'll reopen and continue investigating

**Comment by antico5** [comment]
@vladyan18 @daramir we just released 0.6.5 and the issue should be fixed. Please test and report back. Thanks!

**Comment by vladyan18** [comment]
@antico5 works for me now, thanks!

**Comment by daramir** [comment]
> @vladyan18 @daramir we just released 0.6.5 and the issue should be fixed. Please test and report back. Thanks!

Works for me too on 0.6.6, thanks a lot!
